server/app/views/admin/shared/TemplateGlobals.java

```diff
@@ -0,0 +1,7 @@
+package views.admin.shared;
+
+import lombok.Builder;
+
+/** This record contains globally shared values that are available to any Thymeleaf template. */
+@Builder
```

Comment:
This is lombok. It generates code at compile time, in this case builder methods. It's already part of the project so I figured I'd kick the tires. Definitely liking it more than `autovalue`; at least for something simple.